core/renderers/common/marker_svg.ts

```diff
@@ -572,7 +559,7 @@ export class MarkerSvg {
    *
    * @returns The object holding attributes to make the marker blink.
    */
-  protected getBlinkProperties_(): AnyDuringMigration {
+  protected getBlinkProperties_(): object {
```

Comment:
`object` is definitely more specific than `any`, but might it be worth being more specific still—e.g., defining a suitable record type?

Reply:
I don't think we actually want to be more specific. I think the intention of this is that external developers can override it to provide their own blink properties, which may look different. The code that calls this method doesn't actually inspect the values, it just passes them to `dom.createSvg`.